tools/c7n_azure/tests/test_apimanagement.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from azure_common import BaseTest, arm_template
+
+
+class ApiManagementTest(BaseTest):
+    def setUp(self):
+        super(ApiManagementTest, self).setUp()
+
+    def test_aks_schema_validate(self):

```

Comment:
Four references to AKS in this test file, also cassette name is going to change (or you can rename it).